addOns/sequence/src/main/zapHomeFiles/policies/sequence.policy

```diff
@@ -0,0 +1,313 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<configuration>
+    <policy>sequence</policy>
+    <scanner>
+        <level>OFF</level>
+        <strength>MEDIUM</strength>
+    </scanner>
+    <plugins>
+        <p6>
```

Comment:
Can we add a fake node or commend giving the scan rule name? Just for documentation purposes?
TBH I'd like to make that change to the core as well...